src/vs/platform/remoteTunnel/electron-browser/remoteTunnelService.ts

```diff
@@ -133,8 +133,18 @@ export class RemoteTunnelService extends Disposable implements IRemoteTunnelServ
 			this.setTunnelStatus(TunnelStates.disconnected);
 			return;
 		}
+		const { token, authenticationProviderId } = this._account;
+
 		this.setTunnelStatus(TunnelStates.connecting(localize('remoteTunnelService.authorizing', 'Authorizing')));
-		const loginProcess = this.runCodeTunneCommand('login', ['user', 'login', '--provider', this._account.authenticationProviderId, '--access-token', this._account.token]);
+		const onOutput = (a: string, isErr: boolean) => {
+			a = a.replace(new RegExp(escapeRegExpCharacters(token), 'g'), '*'.repeat(4));
```

Comment:
Can you use `replaceAll` to avoid having to create a regular expression?

Reply:
added 66a27deeef5b0ea8496382fd1c2f1d2341cdfd82